config/secrets.yml

```diff
@@ -0,0 +1,6 @@
+development:
```

Comment:
This is needed because we need to specify secret_key_base in order to share the authentication mechanism with the members' app.